package.json

```diff
@@ -5,40 +5,35 @@
   "private": false,
   "type": "module",
   "source": "src/index.ts",
-  "main": "./dist/scrollrig.cjs",
-  "module": "./dist/scrollrig.module.js",
-  "unpkg": "./dist/scrollrig.umd.js",
   "types": "./dist/index.d.ts",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
   "sideEffects": false,
   "exports": {
     "./css": {
-      "default": "./dist/scrollrig.css"
+      "default": "./dist/style.css"
     },
     "./scrollbar": {
       "types": "./dist/scrollbar/index.d.ts",
-      "require": "./dist/scrollbar.cjs",
-      "import": "./dist/scrollbar.modern.js"
+      "require": "./dist/scrollbar/index.cjs",
+      "import": "./dist/scrollbar/index.js"
     },
     "./powerups": {
-      "require": "./dist/powerups.cjs",
-      "import": "./dist/powerups.modern.js"
+      "types": "./dist/powerups/index.d.ts",
+      "require": "./dist/powerups/index.cjs",
+      "import": "./dist/powerups/index.js"
     },
     ".": {
       "types": "./dist/index.d.ts",
-      "require": "./dist/scrollrig.cjs",
-      "import": "./dist/scrollrig.modern.js"
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js"
     }
   },
   "files": [
-    "dist",
-    "scrollbar",
-    "powerups"
+    "dist"
   ],
   "scripts": {
-    "build": "yarn build-default & yarn build-scrollbar & yarn build-powerups",
-    "build-default": "microbundle --globals @react-three/fiber=@react-three/fiber,@juggle/resize-observer=@juggle/resize-observer,@studio-freight/lenis=@studio-freight/lenis",
-    "build-scrollbar": "microbundle --globals @react-three/fiber=@react-three/fiber,@studio-freight/lenis=@studio-freight/lenis src/scrollbar/index.ts -o dist/scrollbar.js",
-    "build-powerups": "microbundle --globals @react-three/fiber=@react-three/fiber,@studio-freight/lenis=@studio-freight/lenis,@react-three/drei=@react-three/drei,@14islands/r3f-scroll-rig=@14islands/r3f-scroll-rig --external=@14islands/r3f-scroll-rig powerups/index.ts -o dist/powerups.js",
+    "build": "vite build && tsc &&",
     "dev": "microbundle watch"
```

Comment:
I guess the `dev` script should be removed as well since `microbundle` has been removed as a dep. No big deal running build manually.